docpages/01_disdppgloss.md

```diff
@@ -1,51 +1,51 @@
-## A Glossary of Common Discord Terms

```

Comment:
not sure this should come before the faq, maybe after it?

Reply:
okay. ill put it after it

Reply:
had to rename all other pages 🤦